packages/shared/src/recording/metadata/legacy/test/index.ts

```diff
@@ -17,7 +17,7 @@ const versions = {
   ...v2,
 };
 
-export function validate(test: UnstructuredMetadata) {
+export function validate(test?: UnstructuredMetadata) {
   if (!test) {
     throw new Error("Test metadata does not exist");
```

Comment:
This seems like an odd change. Why make the type optional and then throw if it's not passed? 🤔

Reply:
I'm just matching the previous API here to maintain backwards compatibility with this:
https://github.com/replayio/replay-cli/blob/72867d6c6c3f209c58596de1c6cf36d4d0429dea/packages/replay/src/metadata/test.ts

It was a public export. We might not want to publish the old CLI anymore but it's good to have it in a publishable state.

That said, I can look into hoisting this `throw` into that legacy layer.

Reply:
I think it makes sense to keep it at this layer. Essentially those `validate` functions should really accept `unknown` but I don't plan to change the parameter annotation here ;p

Reply:
> Essentially those validate functions should really accept unknown

Yeah, agreed. I was just a bit surprised to see the halfway change :)